src/utils/asyncStorage.js

```diff
@@ -0,0 +1,47 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export async function getMyStringValue (key) {
+  try {
+      return await AsyncStorage.getItem(key)
+  } catch(e) {
+      console.log(e)
+  }
+}
+
+export async function setStringValue (key, value) {
```

Comment:
assuming the value here can be any object, might want to change this function name to be more descriptive

Reply:
It would be another function https://react-native-async-storage.github.io/async-storage/docs/api#clear:~:text=underlying%20storage%20error.-,Example%3A,-getMyStringValue%20%3D%20async

Reply:
makes sense. change it to the object one and i think we will be good here